lib/inspec/plugin/v2/installer.rb

```diff
@@ -284,8 +290,16 @@ def install_from_gem_file(requested_plugin_name, opts)
 
     def install_from_remote_gems(requested_plugin_name, opts)
       plugin_dependency = Gem::Dependency.new(requested_plugin_name, opts[:version] || "> 0")
-      # BestSet is rubygems.org API + indexing
-      install_gem_to_plugins_dir(plugin_dependency, [Gem::Resolver::BestSet.new], opts[:update_mode])
+
+      # BestSet is rubygems.org API + indexing, APISet is for custom sources
+      sources = Gem::Resolver::BestSet.new
+      sources = Gem::Resolver::APISet.new(opts[:source]) if opts[:source]
```

Comment:
One nice way in ruby to implement @clintoncwolfe's suggestion is to do something like

```suggestion
      sources = if opts[:source]
                  Gem::Resolver::APISet.new(opts[:source]) 
                else
                  Gem::Resolver::BestSet.new
                end
```